queue_services/entity-emailer/tests/unit/test_tracker.py

```diff
@@ -123,6 +123,23 @@
                  }
              }
          }),
+        ('f36e3af7-90c3-4859-a6f6-2feefbdc1e30',
+         {
+            'specversion': '1.x-wip',
+            'type': 'bc.registry.dissolution',
+            'source': 'furnishingsJob',
+            'id': 'f36e3af7-90c3-4859-a6f6-2feefbdc1e30',
+            'time': '',
+            'datacontenttype': 'application/json',
+            'identifier': 'BC123223',
+            'data': {
+                'furnishing': {
+                    'type': 'EMAIL',
```

Comment:
`type` should be the `batch_type`.

So `INVOLUNTARY_DISSOLUTION` should be the value we test with.

Reply:
Fixed